package.json

```diff
@@ -49,106 +49,107 @@
     "@codemirror/lang-xml": "^0.20.0",
     "@ember/jquery": "^2.0.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/render-modifiers": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
+    "@ember/render-modifiers": "^2.0.5",
+    "@ember/test-helpers": "^2.9.3",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
     "@kanselarij-vlaanderen/au-kaleidos-css": "3.3.0",
     "@kanselarij-vlaanderen/au-kaleidos-icons": "3.0.1",
     "@lblod/ember-acmidm-login": "git+https://github.com/ValenberghsSven/ember-acmidm-login.git#75d8aab5839585201a68df4701a15bcfda42b32d",
     "@lblod/ember-mock-login": "^0.7.0",
     "@lblod/ember-rdfa-editor": "0.59.1",
     "@lblod/ember-vo-webuniversum-data-table": "^1.0.0",
-    "@ljharb/eslint-config": "^17.1.0",
-    "@typescript-eslint/parser": "^3.8.0",
+    "@ljharb/eslint-config": "^21.0.1",
+    "@typescript-eslint/parser": "^5.48.2",
     "alphabet": "^1.0.0",
     "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
     "broccoli-funnel": "^3.0.8",
     "cypress": "^9.4.1",
     "cypress-file-upload": "^5.0.8",
-    "date-fns": "^2.29.0",
-    "dayjs": "^1.10.7",
-    "ember-auto-import": "^2.0.0",
+    "date-fns": "^2.29.3",
+    "dayjs": "^1.11.7",
+    "ember-auto-import": "^2.5.0",
     "ember-cli": "~3.28.5",
     "ember-cli-app-version": "^5.0.0",
-    "ember-cli-autoprefixer": "^1.0.0",
+    "ember-cli-autoprefixer": "^2.0.0",
     "ember-cli-babel": "^7.26.11",
     "ember-cli-cjs-transform": "^2.0.0",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-dependency-lint": "^1.1.3",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-dependency-lint": "^2.0.1",
     "ember-cli-head": "^2.0.0",
-    "ember-cli-htmlbars": "^6.0.1",
+    "ember-cli-htmlbars": "^6.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-sass": "^8.0.1",
+    "ember-cli-sass": "^11.0.1",
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "^2.1.1",
     "ember-cli-string-helpers": "^6.1.0",
     "ember-cli-terser": "^4.0.2",
-    "ember-cli-update": "^1.0.0",
-    "ember-click-outside": "^2.0.0",
+    "ember-cli-update": "^1.0.1",
+    "ember-click-outside": "^5.0.1",
     "ember-code-snippet": "^3.0.0",
-    "ember-composable-helpers": "^4.4.1",
-    "ember-concurrency": "^2.2.1",
+    "ember-composable-helpers": "^5.0.0",
```

Comment:
New version of ember-composable-helpers has renamed the `contains` helper to `includes`. This was deprecated for a while but now it outright doesn't exist anymore. We don't use it in the app, it's pulled in by `ember-vo-webuniservum-data-table`. I *think* that if you merge development again into this branch it should just work, because we don't use DataTable anywhere then. Currently checking locally if that's the case.